doc/rados/configuration/filestore-config-ref.rst

```diff
@@ -302,7 +302,7 @@ Misc
 
 ``filestore split multiple``
 
-:Description:  ``filestore_split_multiple * abs(filestore_merge_threshold) * 16`` 
+:Description:  ``(filestore_split_multiple * abs(filestore_merge_threshold) + (rand() % filestore_split_rand)) * 16``
```

Comment:
s/filestore_split_rand/filestore_split_rand_factor/

btw, it's a little weird to me that we have both options with **split** prefix and **merge** prefix here...